piperci_gman/resource.py

```diff
@@ -19,34 +19,100 @@ def InternalError(self, message=None):
         message = f': {message}' if message else ''
         return ({'message': f'Internal Server Error{message}'}, 500)
 
-    def get_task_states(self, events):
+    def task_states(self, events):
```

Comment:
We had previously talked about add the event to the pending list instead of the task. Was it decided that adding the task instead is the better option?

Reply:
I was able to add the parent task and track everything via the new parent_id field in the child events. So now all lists being returned from this method are a list of tasks

Reply:
Oh perfect!